### PR TITLE
fix cgo build issue

### PR DESCRIPTION
**Pull request by yanjunding**
- fix cgo compile
- update to geonet `go-scratch` image

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -17,39 +17,39 @@ fi
 
 # code will be compiled in this container
 BUILDER_IMAGE='quay.io/geonet/golang:1.13.1-alpine'
-RUNNER_IMAGE='quay.io/geonet/alpine:3.10'
-
+RUNNER_IMAGE='quay.io/geonet/go-scratch:latest'
+DOCKERFILE=${DOCKERFILE:-"Dockerfile"}
```
Comment (junghao) [inline build.sh:21]: This makes local build/test somehow inconvenient (we can simply type `./build.sh` to build for local test) .

The existing way is good enough.

**Review by junghao — CHANGES_REQUESTED** [review]

File: Dockerfile.cgo [hunk 2]
```diff
@@ -23,21 +23,18 @@ ENV GOPATH /usr/src/go
 ENV GOFLAGS -mod=vendor
 ENV GOOS linux
 ENV GOARCH amd64
-ENV CGO_ENABLED ${CGO_ENABLED}
 
 RUN echo 'nobody:x:65534:65534:Nobody:/:\' > /passwd
 RUN make -B -C vendor/github.com/GeoNet/kit/cvendor/libmseed/ && \
   make -B -C vendor/github.com/GeoNet/kit/cvendor/libslink && \
-  go install -a -installsuffix cgo -ldflags "-X main.Prefix=${BUILD}/${GIT_COMMIT_SHA}" /repo/cmd/${BUILD}
+  CGO_ENABLED=1 go install -a -installsuffix cgo -ldflags "-X main.Prefix=${BUILD}/${GIT_COMMIT_SHA} -extldflags -static" /repo/cmd/${BUILD}
```
[the remark this answers is not in the document]
Comment (yanjunding) [reply Dockerfile.cgo:30]: > If ${BUILD} works then why don't we do CGO_ENABLED=${CGO_ENABLED} or CGO_ENABLED="${CGO_ENABLED}}" both works?
It doesn't work that way.

File: build.sh (hunk 1 above)
Comment (yanjunding) [reply build.sh:21]: @ardrigh how do you think?

File: cmd/fdsn-quake-consumer/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+ENV GOBIN /repo/gobin
+ENV GOPATH /usr/src/go
+ENV GOFLAGS -mod=vendor
+ENV GOOS linux
+ENV GOARCH amd64
+
+RUN echo 'nobody:x:65534:65534:Nobody:/:\' > /passwd
+RUN CGO_ENABLED=0 go install -a -installsuffix cgo -ldflags "-X main.Prefix=${BUILD}/${GIT_COMMIT_SHA}" /repo/cmd/fdsn-quake-consumer
+
+FROM quay.io/geonet/alpine:3.9
```
Comment (junghao) [inline cmd/fdsn-quake-consumer/Dockerfile:25]: I think it's safe to use alpine:3.10.

File: .travis.yml [hunk 4]
```diff
@@ -9,45 +9,66 @@ services:
...
+        bash build.sh
+        fdsn-ws
+        fdsn-holdings-consumer
+        fdsn-slink-db
+        fdsn-quake-consumer
+        fdsn-ws-nrt
+    deploy:
+      - provider: script
+        script: >-
+          baash push.sh
```
Comment (junghao) [inline .travis.yml:64]: typo. `bash`

**Review by junghao — CHANGES_REQUESTED** [review]
Some minor issues.

File: cmd/fdsn-quake-consumer/Dockerfile (hunk 3 above)
Comment (yanjunding) [reply cmd/fdsn-quake-consumer/Dockerfile:25]: keep it alpine or change to go-scratch?

**Review by junghao — APPROVED** [review]
Looks good.

**Pull request merged**